test/IDE/reconstruct_type_from_mangled_name.swift

```diff
@@ -64,22 +64,22 @@ class Myclass2 {
 
     arr1.append(1)
 // FIXME: missing append()
-// CHECK: dref: FAILURE	for 'append' usr=s:Sa6appendyyxF
-// CHECK: type: (inout Array<Int>) -> (Int) -> ()
```

Comment:
@nkcsgexi @benlangmuir not entirely sure what this was testing so the fix here was just to make it match the new output... `Array.append(_:)` now takes its argument owned so the mangled name changed but not sure about the following line.

Reply:
The type change from `(inout Array<Int>) -> (Int) -> ()` to `(inout Array<Int>) -> () -> ()` is bogus.  I assume that means we aren't handling owned et. al. correctly in type reconstruction. @slavapestov what do you think?

Reply:
So would it be better to XFAIL this one for now?

Reply:
You can go ahead and XFAIL it, please do file a radar though.